codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttMessage.java

```diff
@@ -61,7 +54,7 @@ public Object variableHeader() {
     }
 
     public Object payload() {
```

Comment:
doesn't this change in general break backwards compatibility? `MqttMessage` is public and `payload()` now will always return `null`. The behavior is different for users who may extend this class.